time timestamp
float32 headsetTime
float32 battery
int32 wirelessSignal
int32[] contactQuality
